pkgs/development/libraries/libdigidocpp/default.nix

```
@@ -19,6 +19,14 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" "dev" "bin" ];
 
+  # Cherry-pick of
+  # https://github.com/open-eid/libdigidocpp/commit/2b5db855ba3ceb9bae1f11589ea1aea22bb7595a
+  # Fixes https://github.com/NixOS/nixpkgs/issues/334397
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace 'TSA_URL "http://dd-at.ria.ee/tsa"' 'TSA_URL "https://eid-dd.ria.ee/ts"'
```

Not sure if `warn` or `fail` is best. Not blocking either way

```suggestion
      --replace-warn 'TSA_URL "http://dd-at.ria.ee/tsa"' 'TSA_URL "https://eid-dd.ria.ee/ts"'
```

---

Will update to `replace-fail`. This manual substitution should be removed once we bump to a new release.

---

Done